src/stcal/jump/twopoint_difference.py

```diff
@@ -209,32 +209,49 @@ def find_crs(dataa, group_dq, read_noise, normal_rej_thresh,
                     # find_jumps to use when it reconstructs the full group dq
                     # array from the slices.
                     if cr_row[j] != 0:
-                        gdq[integ, cr_group[j], cr_row[j] - 1, cr_col[j]] =\
-                            np.bitwise_or(gdq[integ, cr_group[j], cr_row[j] -
-                                          1, cr_col[j]], dqflags["JUMP_DET"])
+
+                        if (gdq[integ, cr_group[j], cr_row[j] - 1, cr_col[j]] \
```

Comment:
Should probably add a comment in the code here to indicate that we only want to flag neighbors if they aren't saturated and do_not_use.